package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "test-repo",
+  "version": "1.0.0",
```

Comment:
package.json version should match repository version.

Reply:
Can't add string `master` as version in package.json. As per [package.json format](https://docs.npmjs.com/files/package.json#version):

> Version must be parseable by [node-semver](https://github.com/isaacs/node-semver)

Should I raise a separate issue for this? The version file should probably be renamed to `.branch` or similar.